tests/testthat/test-visr_plot.R

```diff
@@ -332,6 +333,27 @@ testthat::test_that("T2.15 An error when an undefined option is passed to `legen
   
 })
 
+testthat::test_that("T2.16 When both `x_label` and `x_units` are specified, they are concatenated into the x-axis label.", {
```

Comment:
add tests for one of them not specified and a test for both not specified

Reply:
T2.1 and T2.3 cover this behaviour since their default is `NULL`.